policyengine_us/parameters/gov/contrib/states/ny/wftc/amount/max.yaml

```
@@ -0,0 +1,19 @@
+description: New York provides the following maximum Working Families Tax Credit amount.
```

Same for minimum IIUC
```suggestion
description: New York provides the following maximum Working Families Tax Credit amount per child.
```